d3a_api_client/redis_client_base.py

```diff
@@ -69,8 +69,9 @@ def _subscribe_to_response_channels(self, pubsub_thread=None):
         channel_subs[f'{self._channel_prefix}/events/tick'] = self._on_tick
         channel_subs[f'{self._channel_prefix}/events/trade'] = self._on_trade
         channel_subs[f'{self._channel_prefix}/events/finish'] = self._on_finish
+        channel_subs[f'{self._channel_prefix}/*'] = self._on_event_or_response
```

Comment:
Subscribe to pattern channels (all of them) and call the `_on_event_or_response`